customScripts/expansionfarm.gd

```diff
@@ -7,9 +7,9 @@ extends Node
 var settings = {
 	#---Farm Worker Determination
 	#Skill: Most Skilled Chosen / Energy: Most Energy Chosen / Equal: Cycles Through All Available
-	farmhand = 'equal',
-	milkmaid = 'equal',
-	stud = 'equal',
+	herders = 'equal',
```

Comment:
This entire file appears to be a revert and undoes various fixes.

Reply:
fixed for next commit